Sources/StreamChat/ChatClient_Tests.swift

```diff
@@ -1340,10 +1349,20 @@ extension ChatClient_Tests {
             return
         }
         
+        var expectedIdentifier: String {
+            #if canImport(StreamChatSwiftUI)
+            "swiftui"
+            #elseif canImport(StreamChatUI)
+            "uikit"
+            #else
+            "swift"
+            #endif
+        }
```

Comment:
Are we explicitly hardcoding the behaviour within the test target? I thought the intention of these tests was to verify the correct value is propagated into the header field.

If we need to keep this, let's turn this into a computed var of this test case to avoid code duplication.

Reply:
References to `StreamChatUI/SwiftUI` look a bit weird in test target for LLC. tbh. Isn't it the matter of clearing the cache by removing derived data?

Reply:
Not really. Even if it did, we can't afford to clear cache every time we want to run tests. Either we remove this piece of tests or we hardcode the behavior, or convert the check so it `startsWith("stream-swift")`

Reply:
Ok, I see your point. Then let's address the comment from Boris about code duplication 👍

Reply:
Ill keep track of this change. We will need to think the strategy through